browser/src/core/LOUtil.js

```diff
@@ -148,6 +164,7 @@ L.LOUtil = {
 				return this.getURL('images/dark/' + imgName);
 			}
 		}
+		defaultImageURL = this.isDarkModeItem(imgName) ? 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=' : defaultImageURL;
```

Comment:
Can we add "label" here what is that base64 image?
`var dummyEmptyImg = 'data:image...';`
It will be more obvious :)